runtime/moonbase/src/weights/pallet_message_queue.rs

```diff
@@ -138,7 +138,13 @@ impl<T: frame_system::Config> pallet_message_queue::WeightInfo for WeightInfo<T>
 			.saturating_add(T::DbWeight::get().reads(2_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
-	/// Storage: `MessageQueue::BookStateFor` (r:1 w:1)
+
+    fn set_service_head() -> Weight {
+        // TODO: regenerate this file
+        <()>::set_service_head()
```

Comment:
Lets try to generate this locally, just to be sure the benchmarks are not broken